helpers/controllers/userInfoController.ts

```diff
@@ -29,8 +31,23 @@ export const userInfo = async (_parent: void, args: Username) => {
       where: {
         userId: user.id
       }
+    }),
+    Star.findAll({
+      where: {
+        mentorId: user.id
+      }
     })
   ])
+  const starMap = starsReceived.reduce((map: any, star: any) => {
+    map[star.lessonId] = map[star.lessonId]?.concat(star.dataValues) || [
```

Comment:
I think it would be more contextual to use `push` over `concat` in this instance because what you are doing is pushing a new element to the `map[star.lessonId]` array. It would also be more efficient since concat will make a loop through the array, while push will not